pyproject.toml

```diff
@@ -1,261 +1,266 @@
-interp_grid = "geoips.plugins.modules.interpolators.scipy_wrappers.interp_grid"
-
-# sed -i "s,pmw_tb.cmap_150H,pmw_150H,g" plugins/yaml/product*/*
-# sed -i "s,pmw_tb.cmap_37H_Legacy,pmw_37H_Legacy,g" plugins/yaml/product*/*
-# sed -i "s,pmw_tb.cmap_37H_Physical,pmw_37H_Physical,g" plugins/yaml/product*/*
-# ...
-# sed -i "s,tpw.tpw_cimss,tpw_cimss,g" plugins/yaml/product*/*
-# ...
-# sed -i "s,visir.Infrared,Infrared,g" plugins/yaml/product*/*
-# sed -i "s,visir.IR_BD,IR_BD,g" plugins/yaml/product*/*
-# sed -i "s,visir.WV,WV,g" plugins/yaml/product*/*
-# sed -i "s,winds.wind_radii_transitions,wind_radii_transitions,g" plugins/yaml/product*/*
-[project.entry-points."geoips.colormappers"]
+[tool.poetry.plugins."geoips.colormappers"]
+IR_BD = "geoips.plugins.modules.colormappers.visir.IR_BD"
+Infrared = "geoips.plugins.modules.colormappers.visir.Infrared"
+WV = "geoips.plugins.modules.colormappers.visir.WV"
 cmap_rgb = "geoips.plugins.modules.colormappers.cmap_rgb"
 matplotlib_linear_norm = "geoips.plugins.modules.colormappers.matplotlib_linear_norm"
 pmw_150H = "geoips.plugins.modules.colormappers.pmw_tb.cmap_150H"
+pmw_37H = "geoips.plugins.modules.colormappers.pmw_tb.cmap_37H"
 pmw_37H_Legacy = "geoips.plugins.modules.colormappers.pmw_tb.cmap_37H_Legacy"
 pmw_37H_Physical = "geoips.plugins.modules.colormappers.pmw_tb.cmap_37H_Physical"
-pmw_37H = "geoips.plugins.modules.colormappers.pmw_tb.cmap_37H"
 pmw_37pct = "geoips.plugins.modules.colormappers.pmw_tb.cmap_37pct"
+pmw_89H = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89H"
+pmw_89HW = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89HW"
 pmw_89H_Legacy = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89H_Legacy"
 pmw_89H_Physical = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89H_Physical"
-pmw_89H = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89H"
 pmw_89pct = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89pct"
-pmw_89HW = "geoips.plugins.modules.colormappers.pmw_tb.cmap_89HW"
 pmw_Rain = "geoips.plugins.modules.colormappers.pmw_tb.cmap_Rain"
 tpw_pwat = "geoips.plugins.modules.colormappers.tpw.tpw_pwat"
-Infrared = "geoips.plugins.modules.colormappers.visir.Infrared"
-IR_BD = "geoips.plugins.modules.colormappers.visir.IR_BD"
-WV = "geoips.plugins.modules.colormappers.visir.WV"
 wind_radii_transitions = "geoips.plugins.modules.colormappers.winds.wind_radii_transitions"
 
-[project.entry-points."geoips.filename_formatters"]
+[tool.poetry.plugins."geoips.coverage_checkers"]
+center_radius = "geoips.plugins.modules.coverage_checkers.center_radius"
+center_radius_rgba = "geoips.plugins.modules.coverage_checkers.center_radius_rgba"
+masked_arrays = "geoips.plugins.modules.coverage_checkers.masked_arrays"
+numpy_arrays_nan = "geoips.plugins.modules.coverage_checkers.numpy_arrays_nan"
+rgba = "geoips.plugins.modules.coverage_checkers.rgba"
+windbarbs = "geoips.plugins.modules.coverage_checkers.windbarbs"
```

Comment:
Were there any updates in the plugins in pyproject.toml since this was branched?  @jsolbrig

Reply:
I went through them by hand. I grabbed the version from `main`, sorted each section, then compared against the new pyproject.toml.

Like with the dependencies, it would be great if you could double-check Kumar's and my work.

Reply:
adding from what I have observed since branched: there was a nomenclature change like coverage_check to coverage_checkers both in namespace and specific plugin. There were others too; all being updated.